lib/internal/Magento/Framework/ObjectManager/Code/Generator/Repository.php

```diff
@@ -228,7 +228,6 @@ protected function _getGetMethod()
             'parameters' => [
                 [
                     'name' => 'id',
-                    'type' => 'int',
```

Comment:
How?


As stated in the commits this type configuration was completely ignored in `zend-code` 2.x see: https://github.com/zendframework/zend-code/blob/master/doc/book/migration.md#string-int-float-bool-are-no-longer-ignored. So removing this should have no affect at all. This now generates scalar type hints in `3.x`.